src/main/java/kubatech/mixin/mixins/minecraft/StringTranslateMixin.java

```diff
@@ -0,0 +1,33 @@
+package kubatech.mixin.mixins.minecraft;
+
+import static kubatech.mixin.MixinsVariablesHelper.currentlyTranslating;
+
+import java.util.regex.Matcher;
+
+import net.minecraft.util.StringTranslate;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import kubatech.Tags;
+
+@SuppressWarnings("unused")
+@Mixin(value = StringTranslate.class)
+public class StringTranslateMixin {
+
+    @Redirect(
+        method = "Lnet/minecraft/util/StringTranslate;parseLangFile(Ljava/io/InputStream;)Ljava/util/HashMap;",
```

Comment:
`Unnecessary qualified reference to 'parseLangFile' in target class`
```suggestion
        method = "parseLangFile(Ljava/io/InputStream;)Ljava/util/HashMap;",
```

Reply:
Done